src/cozmo/action.py

```diff
@@ -189,17 +189,17 @@ def _recv_msg_robot_completed_action(self, evt, *, msg):
             logger.warn('Received "running" action notification for action=%s', self)
             self._set_failed('running', 'Action was still running')
 
-        elif result == types.NOT_STARTED:
+        elif result == types.FAILURE_NOT_STARTED:
```

Comment:
Looks like it changed back for what ever reason
https://github.com/anki/cozmo-one/blob/master/clad/src/clad/types/actionTypes.clad